packages/node/src/node/server/InteractionServer.ts

```diff
@@ -507,7 +519,7 @@ export class InteractionServer implements ProtocolHandler, InteractionRecipient
                 error instanceof MatterError ? error.message : error,
             );
             if (error instanceof StatusResponseError && !(error instanceof ReceivedStatusResponseError)) {
-                logger.info(`Sending status response ${error.code} for interaction error: ${error.message}`);
+                logger.info(`Status response » ${error.code} for interaction error: ${error.message}`);
```

Comment:
Might replace "for interaction error: " with just "Error:".

For bare errors without traces I generally try to format the same as we do with a stack trace.  Should make a `Diagnostic.errorMessage(error: unknown)` or something but for now it looks like I've just used `Diagnostic.squash("[", Diagnostic.strong(error.id), "]"), error.message`

Reply:
adjusted